### PR TITLE
Rinkeby and Mainnet Deployed Versions

### PR DESCRIPTION
**Pull request by ejwessel**
## Description

Quick description:

- [X] N/A
- [ ] New features
- [ ] Bug fixes
- [ ] Typo/small tweaks

## Changes

Implementation details. These are mainly to help your fellow developers.

### CODE REVIEW

File: source/delegate-factory/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@airswap/delegate-factory",
-  "version": "1.4.7",
+  "version": "1.4.8",
```
Comment (poiresel) [inline source/delegate-factory/package.json]: since a major release update major version.

File: source/indexer/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@airswap/indexer",
-  "version": "1.5.7",
+  "version": "1.5.8",
```
Comment (poiresel) [inline source/indexer/package.json]: since deployed to mainnet update the major version.

Comment (ejwessel) [reply source/indexer/package.json]: oh yeah....

File: source/delegate-factory/package.json (hunk 1 above)
Comment (ejwessel) [inline source/delegate-factory/package.json]: 2.5.7

File: source/indexer/package.json (hunk 2 above)
Comment (ejwessel) [inline source/indexer/package.json]: 2.6.7

**Pull request merged**